chocolatey/tools-offline/chocolateyinstall.ps1

```diff
@@ -21,3 +21,11 @@ $packageArgs = @{
   validExitCodes= @(0, 3010, 1641)
 }
 Install-ChocolateyInstallPackage @packageArgs
+
+$installInfo = @"
+tool: chocolatey
+tool_version: chocolatey-$($env:CHOCOLATEY_VERSION)
+installer_version: $($env:chocolateyPackageVersion)-offline
```

Comment:
What's the format of `$env:chocolateyPackageVersion`? If it's just a version number, we might want to prefix it with `chocolatey-` to follow the same format as in other places.